fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMutuallyByRecords.java

```diff
@@ -310,7 +312,7 @@ private CompletableFuture<Void> buildMultiTargetIndex(@Nonnull SubspaceProvider
                             .thenApply(ignore -> null);
                 }, false, null);
 
-        final List<Object> additionalLogMessageKeyValues = Arrays.asList(LogMessageKeys.CALLING_METHOD, "mutualMultiTargetIndex",
+        final List<Object> additionalLogMessageKeyValues = Arrays.asList(LogMessageKeys.CALLING_METHOD, "mutualMultiTargetIndex-wrapper",
```

Comment:
Not a big deal, since the `CALLING_METHOD`s are unique, but it might be clearer to have this one be `IndexingMutuallyByRecords.buildMultiTargetIndex`, and the one below be `IndexingMutuallyByRecords.buildNextRangeOnly`.